packages/connect-explorer/src/js/components/fields/Row.tsx

```diff
@@ -0,0 +1,6 @@
+import styled from 'styled-components';
+
+export const Row = styled.div`
+    margin-bottom: 8px;
+    // min-height: 40px;
```

Comment:
If this commented line is not useful we might want to get rid of it.

Reply:
removed